predicators/utils.py

```diff
@@ -2359,6 +2360,14 @@ def get_env_asset_path(asset_name: str, assert_exists: bool = True) -> str:
     return path
 
 
+def get_third_party_path() -> str:
```

Comment:
nice -- can you make an issue that we should use this everywhere after this is merged?

Reply:
I'll make it now before I forget!